ee/clickhouse/sql/events.py

```diff
@@ -44,30 +46,28 @@
 {storage_policy}
 """
 ).format(
-    table_name=EVENTS_TABLE,
-    cluster=CLICKHOUSE_CLUSTER,
-    engine=table_engine(EVENTS_TABLE, "_timestamp", REPLACING_MERGE_TREE),
+    table_name=EVENTS_DATA_TABLE(),
+    cluster=settings.CLICKHOUSE_CLUSTER,
+    engine=ReplacingMergeTree("events", ver="_timestamp", replication_scheme=ReplicationScheme.SHARDED),
```

Comment:
nit, non blocker: this was confusing to me at first - I suspected that the class would account for ClickHouse replication - and it does - but it is a bit misleading that I pass in `ReplicationScheme.SHARDED` but it might actually be `ReplicationScheme.NOT_SHARDED`.

Reply:
Suggestions on how to clear this up?

Reply:
Do the env check here instead of down in the class? Does move a bit of clutter elsewhere though at the benefit of being more explicit.

I wonder if there's also a more explicit name for the argument that would help clarify this.

Reply:
Going to do so in a follow-up as my subsequent commits would otherwise conflict. Thanks for the ideas! :)